pages.es/common/type.md

```diff
@@ -0,0 +1,21 @@
+# Tipo
+
+> Muestra el tipo de comando que ejecutará el intérprete de comandos.
+> Nota: todos los ejemplos no son compatibles con POSIX.
+> Más información: <https://manned.org/type>.
+
+- Muestra el tipo de un comando:
+
+`type {{comando}}`
+
+- Muestra todas las ubicaciones que contienen el ejecutable especificado (sólo funciona en los intérpretes de comandos Bash/fish/Zsh):
+
+`type -a {{comando}}`
+
+- Muestra el nombre del archivo de disco que se ejecutaría (sólo funciona en intérpretes de comandos Bash/fish/Zsh):
```

Comment:
```suggestion
- Muestra el nombre del archivo en disco que se ejecutaría (solo funciona en intérpretes de comandos Bash/fish/Zsh):
```

What do you think about using the preposition `en` instead of `de`, with the former I think it is clear that the file resides in the user's disk.